### PR TITLE
test: enable pytest to automatically discover tests

### PR DESCRIPTION
**Pull request by Xiretza**
By default, pytest only looks for modules named `test_*`.

### CODE REVIEW

File: .github/workflows/Pipeline.yml [hunk 1]
```diff
@@ -215,4 +215,4 @@ jobs:
           . ./.github/scripts/activate.sh
 
           pip3 install -r ./test/requirements.txt
-          pytest -vsrA --color=yes test/wrappers.py
+          pytest --verbose --capture=no -rA --color=yes
```
Comment (umarcor) [inline .github/workflows/Pipeline.yml:218]: Please, keep subdir `test` as the context. It's ok to discover all the files/tests in that subdir, but we don't want pytest to pick other content in the repo.

**Comment by umarcor** [comment]
I'm curious about the `--capture=no`. I need to leave now, so I'll read about it later. Nonetheless, I'd be glad if you could provide a brief explanation of the rationale.

**Comment by Xiretza** [comment]
That's just the expanded version of `-s`. Which is exactly why I'm expanding it, the short option is just way too easy to miss :)

**Review by umarcor — APPROVED** [review]
LGTM!

**Pull request merged**